rental.rb

```diff
@@ -0,0 +1,14 @@
+require './book'
+
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] for this relationship to work Getters and Setters for @book and @person instance variables should be present